charts/pega/templates/_helpers.tpl

```diff
@@ -130,6 +130,9 @@ until cqlsh -u {{ $cassandraUser | quote }} -p {{ $cassandraPassword | quote }}
 # Additional JVM arguments
 - name: JAVA_OPTS
   value: "{{ .node.javaOpts }}"
+# Additional CATALINA arguments
+- name: CATALINA_OPTS
+  value: "{{ .node.catalinaOpts }}"
```

Comment:
catalinaOpts is not present in actual values.yaml and might result in nil reference during deployment. Can we validate that or look for node.catalinaOpts before assigning to an ENV only if present.

Reply:
Added the validation check for catalinaOpts